app/components/workflows/operations-sidebar.js

```diff
@@ -2,15 +2,43 @@ import Ember from 'ember';
 import Ps from 'npm:perfect-scrollbar';
 
 export default Ember.Component.extend({
+
+  filterText: '',
+  store: Ember.inject.service('store'),
+
+  init() {
+    this._super(...arguments);
+    this.addObserver('filterText', this, 'filterDidChange');
```

Comment:
There is a better way to do this. You can use a computed property so you don't need to add this observer and implement the function below: https://guides.emberjs.com/v2.14.0/object-model/computed-properties/